pkg/driver/controller.go

```diff
@@ -165,12 +167,18 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 				fsOptions.SubnetIds = util.SplitCommasAndRemoveOuterBrackets(value)
 			case volumeParamsTags:
 				fsOptions.Tags = util.StringToStringPointer(value)
-			case volumeParamsSkipFinalBackup:
+			case volumeParamsOptionsOnDeletion:
+				fsOptions.OptionsOnDeletion = util.StringToStringPointer(value)
+			case volumeParamsSkipFinalBackupOnDeletion:
 				boolVal, err := util.StringToBoolPointer(value)
 				if err != nil {
-					return nil, status.Errorf(codes.InvalidArgument, "invalid parameter of %s: %s", volumeParamsSkipFinalBackup, err)
+					return nil, status.Errorf(codes.InvalidArgument, "invalid parameter of %s: %s", volumeParamsSkipFinalBackupOnDeletion, err)
 				}
-				fsOptions.SkipFinalBackup = boolVal
+				fsOptions.SkipFinalBackupOnDeletion = boolVal
+			case volumeParamsVolumeType:
+				continue
+			default:
+				return nil, status.Errorf(codes.InvalidArgument, "invalid parameter of %s: %s", key, value)
```

Comment:
I would love to have this but we can't. Aside from the parameters we define, other reserved variables like `csi.storage.k8s.io/provisioner-secret-name` is returned (see https://kubernetes-csi.github.io/docs/external-provisioner.html). Unless all these reserved parameters are stored somewhere we can't implement this without false errors. We shouldn't manually enter them either.

This reminds me that we should add this to our FAQ, it's possible to enter nonexistent parameters and not error.

Reply:
How do you feel about ignoring but not erroring on parameters that have the csi.storage.k8s.io prefix?

Reply:
Sounds like a solution